bootstrap/backends/ldap/ldap.go

```diff
@@ -87,9 +91,24 @@ func NewIdentityManager(bs bootstrap.Bootstrap) (identity.Manager, error) {
 		subMapping = strings.Split(subMappingString, " ")
 	}
 
+	// Use a clone here to avoid changing the config of other possible users of the config.
+	tlsConfig := config.TLSClientConfig.Clone()
+	if caCertFile := os.Getenv("LDAP_TLS_CACERT"); caCertFile != "" {
+		if pemBytes, err := ioutil.ReadFile(caCertFile); err == nil {
+			rpool, _ := x509.SystemCertPool()
+			if rpool.AppendCertsFromPEM(pemBytes) {
+				tlsConfig.RootCAs = rpool
+			} else {
+				return nil, fmt.Errorf("Error appending CA certificate(s) '%s' to pool", caCertFile)
```

Comment:
error strings should start lower case, to be in line with the other error messages in lico, use something like `failed to append CA certificates from '%s' to tool`.